packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -217,12 +217,10 @@ const CFDsListing = observer(() => {
                 <React.Fragment>
                     {combined_cfd_mt5_accounts.map((existing_account, index: number) => {
                         const list_size = combined_cfd_mt5_accounts.length;
-                        const track_account_name = is_demo
-                            ? `${existing_account.name} ${localize('Demo')}`
-                            : existing_account.name;
+                        const track_account_name = is_demo ? `${existing_account.name} Demo` : existing_account.name;
```

Comment:
this variable is not being used anywhere, do we still need this anywhere?

Reply:
Thanks @shahzaib-deriv